### DISCUSSION TITLE
Firefox userChrome.css script not working as expected in Floorp [Bug]

### DISCUSSION THREAD

**Discussion by Faelights** [Floorp-Projects/Floorp#640] [Q&A]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

I'm not sure if this is the place to ask about this, so please feel free to close this issue if it's out of scope.

I am trying to switch over from FF to Floorp but have found that the CSS script that I use for auto-hiding FF's native sidebar (NOT the Browser Manager Sidebar) is not working as expected in Floorp when the sidebar is displayed on the right.

When I hover over the sidebar area, the sidebar un-hides itself by extending right instead of left and the visible portion is cut off by the edge of the browser.

It still works as expected when the sidebar is displayed on the left.

### To Reproduce

1. Add the following script to userChrome.css file in Floorp's chrome folder for the profile and save.
[sidebar autohide.txt](https://github.com/Floorp-Projects/Floorp/files/13743148/sidebar.autohide.txt)

2. Start browser to load userChrome.css.
3. Make sure sidebar is active and set to display on the right - sidebar should be in collapsed/hidden state.
4. Hover over right edge of browser to unhide sidebar.

### Expected behavior

Sidebar un-hides and extends left over the browser from the right edge of browser on mouse hover.

### Screenshots

![image](https://github.com/Floorp-Projects/Floorp/assets/4390826/f697bc1b-f3b0-436b-b50c-efdfb7cf5198)

![image](https://github.com/Floorp-Projects/Floorp/assets/4390826/d0a7b6e4-8d7e-40cc-9b83-d43a46b74322)

### OS Info

Windows 10 Home 22H2 19045.3803

### Floorp Version

11.7.1

**Comment by Ryosuke-Asano** [comment]
This topic should be talked on discussion. Migrate to it.

**Comment by Not-A-Coder2023** [comment]
This happens to me with WaveFox

**Reply by Ryosuke-Asano** [reply]
SO.. You should change design to Proton
